src/components/Card.js

```diff
@@ -3,10 +3,24 @@ import React from "react";
 const Card = () => {
   return (
     <div>
+    <CardItem>
       <img src="https://rickandmortyapi.com/api/character/avatar/2.jpeg" alt="Hier muss ein Bild sein" />
       <h2> Morty Smith </h2>
+    </CardItem>
     </div>
   );
 };
 
+
+const CardItem = styled.li`
+
+display: flex;
+align-items: center;
+border: 3px sold black;
+margin: 1em;
+list-style-type: none;
```

Comment:
As you are using `list-style-type: none` it is necessary to add `role="list"` as attribute to your `ul` element for accessibility reasons. Screenreaders can't identify lists as lists if `list-style-type`is `none`.